chsdi/templates/htmlpopup/statistikwasserkraftanlagen_new.mako

```diff
@@ -0,0 +1,183 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% 
+    c['stable_id'] = True
+    lang = lang if lang in ('fr','it') else 'de'
+    hydropowerplanttype = 'hydropowerplanttype_%s' % lang
+%>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_wastanumber')}</td>
+      <td>${c['featureId']}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_name')}</td>
+      <td>${c['attributes']['name']}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_location')}</td>
+      <td>${c['attributes']['location'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_canton')}</td>
+      <td>${c['attributes']['canton'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_hydropowerplanttype')}</td>
+      <td>${c['attributes'][hydropowerplanttype] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_beginningofoperation')}</td>
+      <td>${c['attributes']['beginningofoperation'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('tt_ch.bfe.statistik-wasserkraftanlagen_endofoperation')}</td>
+      <td>${c['attributes']['endofoperation'] or '-'}</td>
+    </tr>
+</%def>
+
+<%def name="extended_info(c, lang)">
+<%
+    c['stable_id'] = True
+    lang = 'de' if lang in ('de', 'rm', 'en') else lang
+    import urllib2
+    has_picture = True
+    image_file = None
+    try:
+        image_file == urllib2.urlopen("http://www.bfe-gis.admin.ch/bilder/ch.bfe.statistik-wasserkraftanlagen/%d.jpg" % c['featureId'])
```

Comment:
actually if you're just testing if the image exists.
use:
h.resource_exists(url)

it uses the requests module and performs a head request to determine the existance of the image.

Implementation can be found in helpers